src/main/java/de/tum/in/www1/artemis/service/hestia/ProgrammingExerciseGitDiffReportService.java

```diff
@@ -158,6 +164,26 @@ public ProgrammingExerciseGitDiffReport getOrCreateReportOfExercise(ProgrammingE
         }
     }
 
+    public ProgrammingExerciseGitDiffReport createReportForSubmissions(long submissionId1, long submissionId2) throws GitAPIException, IOException {
```

Comment:
Can you add Javadoc? Why do we need two submissions here? Is one of them the template submission?

Reply:
No, we calculate the diff between submission A and submission B and both have different IDs, gonna add javadoc to explain that.